external-docs/docs/guides/configuration/code-list-config.md

```diff
@@ -0,0 +1,210 @@
+
+#### Using `same_as`
+
+The `same_as` field allows using a concept defined elsewhere in the internet (e.g. [E92 concept in Geography Linked Data](http://statistics.data.gov.uk/id/statistical-geography/E92000001)).
+
+### Linking the code-list-config.json in qube-config.json
+
+Once the `code-list-config.json` is defined, it can be linked in the `qube-config.json` using the `code_list` field in [dimension configuration](./qube-config.md#dimension-configuration). More specifically, the path to the `code-list-config.json` needs to be given in the `code_list` field in `qube-config.json`.
+## Defining an in-line code list
+
+This approach is recommended for defining simple code lists (e.g. code lists with a small number of concepts or simple hierarchy).
+
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1.1",
+    "title": "Example cube config with an inline code list",
+    "summary": "Cube config with an inline code list",
+    "description": "This is an example cube config demonstrating how to define an inline code list",
+    "license": "https://creativecommons.org/licenses/by/4.0/",
+    "creator": "http://statistics.data.gov.uk",
+    "publisher": "http://statistics.data.gov.uk",
+    "dataset_issued": "2022-04-08T00:00:00Z",
+    "keywords": [
+        "Keyword1"
+    ],
+    "columns": {
+        "DimensionInline": {
+            "type": "dimension",
+            "code_list": {
+                "title": "Example of an inline code list",
+                "description": "This is an inline code list defined inside a cube config",
+                "summary": "This is an inline code list",
+                "creator": "http://purl.org/dc/aboutdcmi#DCMI",
+                "publisher": "http://purl.org/dc/aboutdcmi#DCMI",
+                "dataset_issued": "2022-03-31T12:54:30Z",
+                "dataset_modified": "2022-04-01T12:54:30Z",
+                "license": "https://creativecommons.org/licenses/by/4.0/",
+                "themes": [
+                    "http://example.com/themes/theme"
+                ],
+                "keywords": [
+                    "keyword1"
+                ],
+                "sort": {
+                    "by": "label",
+                    "method": "ascending"
+                },
+                "concepts": [
+                    {
+                        "label": "A",
+                        "description": "A data record",
+                        "notation": "a",
+                        "sort_order": 2,
+                    },
+                    {
+                        "label": "C",
+                        "description": "C data record",
+                        "notation": "c",
+                        "sort_order": 1,
```

Comment:
Valid JSON doesn't have trailing commas after the last property/value pair.

I'm going to change the JSON a little bit so I wouldn't worry about fixing it for now.